docs/source/API/containers/ScatterView.rst

```diff
@@ -96,18 +91,36 @@ Description
        resize a view with discarding old data
 
 
-    .. rubric:: *Private* Members
+Constructors
```

Comment:
I think this constructor subsection should be placed before the public class methods subsection.

Reply:
@ldh4 - [agreed](https://kokkos.org/kokkos-core-wiki/templates/class_api.html)